test/luacpptemplater_gtest_supp.h

```diff
@@ -1,5 +1,5 @@
 #include "gtest/gtest.h"
-#include "util/luacpptemplater_stringhelper.h"
+#include "luacpptemplater_stringhelper.h"
```

Comment:
I figured this was only needed for the unit tests so I moved them there. Maybe under test/util it would be better.